load-test-framework/src/main/java/com/google/pubsub/flic/Driver.java

```diff
@@ -141,6 +154,22 @@
           "message loss. If set less than 1, this flag is ignored."
   )
   private int numberOfMessages = 0;
+  @Parameter(
+      names = {"--max_publish_latency_test"},
+      description = "This sets the maximum latency, in this test we will continuously run load " +
+          "tests with increasing request rates until we hit the provided latency. You must only " +
+          "provide a single type of publisher to use this test. This uses the 99% latency as the " +
+          "bound to check."
+  )
+  private int maxPublishLatency = 0;
+  @Parameter(
+      names = {"--max_subscriber_throughput_test"},
+      description = "This test will continuously run load tests with greater publish request " +
+          "rate until the subscriber can no longer keep up, and will let you know the maximum " +
```

Comment:
Can we parameterize in a flag the "no longer keeping up" condition? Perhaps a size of backlog or an oldest unacked message age.